webapp/channels/src/components/post/index.tsx

```diff
@@ -147,7 +150,9 @@ function makeMapStateToProps() {
             teamName = team?.name || currentTeam?.name;
         }
 
-        const canReply = isDMorGM || (channel.team_id === currentTeam?.id);
+        const burnOnReadFeatureEnabled = isBurnOnReadEnabled(state);
+        const isBurnOnReadPost = burnOnReadFeatureEnabled && post.type === PostTypes.BURN_ON_READ;
```

Comment:
If someone turns off burn on read feature, do revealed posts become invisible? If yes, I can see some value in retaining revealed posts for receivers and senders even after the feature id disabled.

Reply:
this is a very good point. In my opinion a BoR defined message should never change the secret nature of it, so if the config is disabled all these message should not be shown. @isacikgoz what is the handling from the backend perspective?
cc: @abhijit-singh